nano/node/common.cpp

```diff
@@ -207,7 +207,7 @@ status (parse_status::success)
 
 void nano::message_parser::deserialize_buffer (uint8_t const * buffer_a, size_t size_a)
 {
-	static nano::network_params network_params;
+	static nano::network_constants network_constants;
```

Comment:
Is the network_params changes unrelated to this PR and possible to put in a separate PR? Not important, but could reduce the size of this patch a bit if that's the case.